doc/UserGuide/InstallAndBasicUsage.rst

```diff
@@ -48,6 +48,8 @@ run using a ``virtualenv``:
 
 To install pip locally if it's not on your system and you don't have admin access please consult: https://pip.pypa.io/en/stable/installing/
 
+With introduction Ganga GUI, new dependencies are included. ``pip install ganga`` installs all the requirements required for running all the functionalities currently availble for Ganga.
+
```

Comment:
I do not think this comment is required. On the other hand, it would be good to add the line
```bash
python3 -m pip install --upgrade pip setuptools wheel
```
before the `pip install ganga` and also to replace the `pip install` with `python3 -m pip install` in that line.